CanvasCore/CanvasCore/React Native Modules/QLPreviewManager.m

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+#import "QLPreviewManager.h"
+#import <UIKit/UIKit.h>
+#import <CanvasCore/CanvasCore-Swift.h>
+
+@implementation QLPreviewManager
+
+RCT_EXPORT_MODULE();
+
+RCT_EXPORT_METHOD(previewFile:(NSString *)url)
+{
+    _fileURL = [[NSURL alloc] initWithString:url];
```

Comment:
This isn't bad but it's normally preferred to use `self.fileURL` in this kind of situation.